buf/registry/module/v1beta1/tag_service.proto

```diff
@@ -120,6 +126,34 @@ message CreateTagsResponse {
   repeated Tag tags = 1 [(buf.validate.field).repeated.min_items = 1];
 }
 
+message MoveTagsRequest {
+  // An individual request to move a Tag.
+  message Value {
+    // The Module associated with the Tag.
+    ModuleRef module_ref = 1 [(buf.validate.field).required = true];
+    // The Tag name.
+    string name = 2 [
+      (buf.validate.field).required = true,
+      (buf.validate.field).string.max_len = 250
+    ];
+    // The id of the Commit to associate with the Tag.
+    string commit_id = 3 [
```

Comment:
Instead of moduleref+commit, whay about a single resourceref like proposed in #33? Same suggestion for the Create RPC.

Reply:
Yup just did it :)